docs/3.0.0-beta.x/community/contribute-with-docs.md

```diff
@@ -0,0 +1,486 @@
+| NPX, Npx, npx       |      npx      | Run packages without downloading using npx. |
+| PM2, pm2, Pm2       |      pm2      | Guys just installed pm2 on my live server.  |
+| postgressql         |  PostgreSQL   |  Want to learn more about PostgreSQL, ...   |
+| React, react.js     |     React     | React allows you to interface with other... |
+| SqLite, sqlite      |    SQLite     |       SQLite is in the public domain.       |
+| Strapi, Strapi.js   |    Strapi     |   Get ready to get Strapi up and running.   |
+| Vue, VUE, Vue.js    | Vue or Vue.js |  My favorite component library for Vue...   |
+| yarn, YARN, Yarn    |     Yarn      | You can use Yarn to keep it all up to date. |
+
+---
+
+#### Using images
+
+If you use images in the documentation, it is a best practice to include both HTML **alt text** and **title text**. The **alt text** appears in place of a broken link, and the **title text** appears when someone hovers over the image. Both are used to help with _accessibility_.
+
+Markdown example,
+
+```
+![alt-text](/src/of/the/image.png) 'title of image')
+```
+
+The following example generates the following logo.
+
+```
+![The Strapi Logo](../assets/contribution-guide/contribute-with-docs/strapi-logo.png 'Strapi Logo')
+```
+
+#### The Strapi Logo
+
+![The Strapi Logo](../assets/contribution-guide/contribute-with-docs/strapi-logo.png 'Strapi Logo')
+
+---
+
+#### Use an unordered list for action steps, number the large sections
+
+There are several reasons for using an unordered list for the **actionable** steps in the documentation.
+
+1. Many developers do not read the theory and are just looking for the steps they need for a specific action. Bullets are an effective way to show this.
+2. Using unordered lists ensure all **actionable** steps are consistent throughout the docs and make the experience better for developers.
+3. Use ordered lists for explanations or theory in a list format.
+
+See the examples below:
+
+**Example #1**
+
+First, you will set-up a Nuxt.js project.
+
+- Install the **Vue CLI** from your terminal.
+
+```
+yarn global add @vue/cli
+
+# OR
+
+npm install -g @vue/cli
+
+```
+
+**Example #2**
+
+Getting started with a Node.js:
+
+1. You need to have a Node.js server installed
+2. You need to have Git Installed
+
+- Verify the version of Node.js:
+
+```bash
+node -v
+```
+
+- Verify the verion of Git:
+
+```bash
+
+git --version
+
+```
+
+---
+
+#### Indicate the path for commands
+
+Whenever you are asking someone to execute a command or to drop in a file, you indicate the `Path: ./` each time. For example, many tutorials involve going back and forth between the `backend` (Strapi) and `frontend` (Frontend Client), and individuals can get lost as to where to execute commands or install dependencies.
+
+See the examples below:
+
+**Example #1**
+
+- Create and enter a directory named `strapi-tutorial`:
+
+`Path: ./Projects/`
+
+```bash
+
+mkdir strapi-tutorial && cd strapi-tutorial
```

Comment:
Why there is an empty line above, but not below the command. It is inconsistent by looking at the bash commands such as the `git --version` just above.

Reply:
Moreover, is it a good practice to add empty lines?

Reply:
Thank you. I removed any empty lines through the PR. It makes sense to not have empty lines before and after the content INSIDE the code block.